vnext/Microsoft.ReactNative/Views/VirtualTextViewManager.cpp

```diff
@@ -20,28 +24,57 @@ using namespace xaml::Documents;
 namespace Microsoft::ReactNative {
 
 void VirtualTextShadowNode::AddView(ShadowNode &child, int64_t index) {
-  auto view = static_cast<ShadowNodeBase &>(child).GetView();
-
-  if (auto run = view.try_as<xaml::Documents::Run>()) {
-    transformableText.originalText = run.Text().c_str();
-    run.Text(transformableText.TransformText());
-  } else if (auto span = view.try_as<xaml::Documents::Span>()) {
+  auto &childNode = static_cast<ShadowNodeBase &>(child);
+  ApplyTextTransformToNode(childNode, textTransform, /* hasChanged = */false);
+  if (auto span = childNode.GetView().try_as<xaml::Documents::Span>()) {
```

Comment:
see if this can be up-classed

Reply:
This is just re-using whatever behavior was there before to test whether the highlight data (background color) needs to be updated. I don't want to break this here. At any rate, same comment as above. We're using Run vs. Span as a reasonably safe way to distinguish between VirtualTextShadowNode and RawTextShadowNode (safe because users of react-native-windows as of today cannot create ViewManagers that would return classes derived from Inline). Another way to handle this would be to create an InlineTextShadowNode base class to enable this differentiation.

Reply:
Not fixing here as this is not modifying existing behavior.